api/test/integration/listing.e2e-spec.ts

```diff
@@ -661,6 +661,57 @@ describe('Listing Controller Tests', () => {
     });
   });
 
+  describe('duplicate endpoint', () => {
```

Comment:
Nit: Can these be separated to improve readability on if both duplication calls are broken or if it is dependent on the units piece?